### PR TITLE
Update cookies and help footer links

### PR DESCRIPTION
**Pull request by rmccar**
### What is the context of this PR?
This PR is to
- update the cookies link in the footer to not open in a new tab
- reorganise the if statement around the footer links to make it consistent with the one that deals with the links in the accessibility page
- add the help links for the different versions of the census

### How to review 
Launch a census questionnaire and check the footer links are going the correct places for the different versions of the questionnaire (some links will return a 404 because the pages aren't live yet). Also check that the cookies page doesn't open in a new tab but all the others do.

### CODE REVIEW

File: templates/layouts/configs/_footer.html [hunk 1]
```diff
@@ -10,11 +15,7 @@
 
   {% set contact_us_url = "https://cyfrifiad.gov.uk/cysylltu-a-ni/" %}
   {% set cookies_url = "https://cyfrifiad.gov.uk/cwcis/" %}
-
-{% else %}
-
-  {% set contact_us_url = "https://census.gov.uk/contact-us/" %}
-  {% set cookies_url = "https://census.gov.uk/cookies/" %}
+  {% set help_url = "https://cyfrifiad.gov.uk/help/sut-i-ateb-y-cwestiynau/help-y-cwestiynau-ar-lein-ew/" %}
```
Comment (ajmaddaford) [inline templates/layouts/configs/_footer.html:18]: This isn't working if you're on one of the static pages in eQ:
- Start the Welsh household survey in Welsh
- Click on the `/privacy` link (Preifatrwydd a diogelu data)
- The links to Contact Us, Help and Cookies are the English links

File: templates/layouts/configs/_footer.html [hunk 2]
```diff
@@ -1,7 +1,12 @@
+  {% set contact_us_url = "https://census.gov.uk/contact-us/" %}
+  {% set cookies_url = "https://census.gov.uk/cookies/" %}
+  {% set help_url = "https://census.gov.uk/help/how-to-answer-questions/online-questions-help/" %}
```
Comment (ajmaddaford) [inline templates/layouts/configs/_footer.html]: The indentation is off for these three lines.

**Review by ajmaddaford — CHANGES_REQUESTED** [review]

File: templates/layouts/configs/_footer.html (hunk 1 above)
Comment (rmccar) [reply templates/layouts/configs/_footer.html:18]: Ive fixed this now. I for some reason language_code wasn't being set on static pages so I have added it to the context and got it from there

**Review by ajmaddaford — APPROVED** [review]

File: templates/layouts/configs/_footer.html [hunk 3]
```diff
@@ -1,7 +1,12 @@
+{% set contact_us_url = "https://census.gov.uk/contact-us/" %}
+{% set cookies_url = "https://census.gov.uk/cookies/" %}
+{% set help_url = "https://census.gov.uk/help/how-to-answer-questions/online-questions-help/" %}
```
Comment (MebinAbraham) [inline templates/layouts/configs/_footer.html:3]: Can we just make this part of an `else` just so it is not potentially set twice?

```jinja
{% if schema_theme == "census-nisra" %}

  {% set contact_us_url = "https://census.gov.uk/ni/contact-us/" %}
  {% set cookies_url = "https://census.gov.uk/ni/cookies/" %}
  {% set help_url = "https://census.gov.uk/ni/help/how-to-answer-questions/online-questions-help/" %}
  {% set logo = "nisra-logo-black-en" %}
  {% set alt = "Nisra Logo" %}
  {% set class_small_logo = "footer__poweredby-img--small" %}

{% elif language_code == "cy" %}

  {% set contact_us_url = "https://cyfrifiad.gov.uk/cysylltu-a-ni/" %}
  {% set cookies_url = "https://cyfrifiad.gov.uk/cwcis/" %}
  {% set help_url = "https://cyfrifiad.gov.uk/help/sut-i-ateb-y-cwestiynau/help-y-cwestiynau-ar-lein-ew/" %}
  
{% else %}
  
  {% set contact_us_url = "https://census.gov.uk/contact-us/" %}
  {% set cookies_url = "https://census.gov.uk/cookies/" %}
  {% set help_url = "https://census.gov.uk/help/how-to-answer-questions/online-questions-help/" %}

{% endif %}
```

**Review by MebinAbraham — APPROVED** [review]

File: templates/layouts/configs/_footer.html (hunk 3 above)
Comment (rmccar) [reply templates/layouts/configs/_footer.html:3]: So, originally I had it like that above but I had a comment on another PR from Ian who suggested changing it to how it is now. Ill leave it as it is for now, unless anyone else has a problem with it and we can change it in another PR

**Pull request merged**